### PR TITLE
Fixes the prod setup

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- Ensures to executing the ./entrypoint.sh
- Ensures "wait-for-it.sh" is available
- Correct prod-dependencies for the nodejs-environment

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM node:14-buster-slim AS build
 
 RUN apt-get -y update && apt-get -y upgrade && \
-    apt-get install --no-install-recommends -y wait-for-it
+    apt-get install --no-install-recommends -y wait-for-it && \
```
Comment (FinnStutzenstein) [inline Dockerfile:4]:
```suggestion
    apt-get install --no-install-recommends -y wait-for-it git
```

File: Dockerfile [hunk 2]
```diff
@@ -16,19 +17,25 @@ RUN npm ci
 # Application lays in /app/src
 COPY ./auth ./
 
+RUN git clone https://github.com/vishnubob/wait-for-it.git ./scripts
```
Comment (FinnStutzenstein) [inline Dockerfile:20]: Why is this needed? Please do not include unchecked code into the image.

Comment (GabrielInTheWorld) [reply Dockerfile:20]: This "installs" the script "wait-for-it"

Comment (FinnStutzenstein) [reply Dockerfile:20]: But you installed it above via apt-get?

Comment (FinnStutzenstein) [reply Dockerfile:20]: Also before adding foreign unchecked code, you can do something like this: https://github.com/OpenSlides/openslides-backend/blob/master/dev/wait.sh

Comment (GabrielInTheWorld) [reply Dockerfile]: I could do this in another PR
